examples/atari_deepRL_saliency.ipynb

```diff
@@ -711,14 +688,11 @@
     "f = plt.figure(figsize=[8,12])\n",
     "frame_ix = 1070\n",
     "\n",
-    "img = history['ins'][frame_ix].copy().astype('uint16')\n",
+    "img = history['ins'][frame_ix].copy()\n",
     "\n",
-    "# Highlight actor saliency in blue and critic saliency in red\n",
-    "img[35:195,:,2] += postprocess_sal(actor_sal_maps[frame_ix-1020], 300).astype('uint16')\n",
-    "img[35:195,:,0] += postprocess_sal(critic_sal_maps[frame_ix-1020], 600).astype('uint16')\n",
-    "img = img.clip(1,255).astype('uint8')\n",
+    "viz = visualize_saliency(img, actor_sal_maps[frame_ix-1020], critic_sal_maps[frame_ix-1020])\n",
     "\n",
-    "plt.imshow(img)\n",
+    "plt.imshow(viz)\n",
```

Comment:
The saliency here looks much weaker compared to what we had originally, but it could be due to the fact that those visualization tricks were used to help "saturate" the saliency and make it stronger than it actually was. It could also be due to the fact that the frame we're explaining here is different than the original frame (see the comment above about the ATARI environment and seed), which I had cherry-picked to show good saliency.

Reply:
This was indeed caused by the incorrect gym version (see above).

Alpha blending the heatmap with the game frame results in slightly less visible saliency in the visualization. I increased the alpha value of the saliency slightly to make it pop a little more.